src/components/AddItem.jsx

```diff
@@ -4,34 +4,7 @@ import { getDoc, doc, setDoc } from 'firebase/firestore';
 import { db } from '../lib/firebase';
 import { useNavigate } from 'react-router-dom';
 import toast, { Toaster } from 'react-hot-toast';
-
-const radioButtonOptions = [
-  {
-    option: 'soon',
-    value: 7,
-    defaultChecked: true,
-  },
-  {
-    option: 'kind of soon',
-    value: 14,
-    defaultChecked: false,
-  },
-  {
-    option: 'not soon',
-    value: 30,
-    defaultChecked: false,
-  },
-];
-
-function removePunctuation(string) {
-  const punctuationlessString = string
-    .toLowerCase()
-    .replace(/[^\w\s]|_/g, '')
-    .split(' ')
-    .join('');
-
-  return punctuationlessString;
-}
+import { radioButtonOptions, removePunctuation } from '../utils';
```

Comment:
- good naming on removePunctuation. IDEs these days are advanced enough that hovering over user will see that it works with strings.
- if this was bigger app, removePunctuation would belong in something like utilsString.js, but no need for this smaller app
- radioButtonOptions is a great way of thinking, but I wouldn't put it in utilities. I'd make another file called appSettings or appConfiguration or configuration.js or such, so it's obvious that file configures the app. Plus, you can put other app settings there too! 
- to be consistent if abbreviation is used or not (button instead of btn, select instead of sel, and so on), utils should be named 'utilities'